src/Sulu/Bundle/AdminBundle/Resources/js/utils/TestHelper/findWithHighOrderFunction.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type {Component} from 'react';
+
+export default function findWithHighOrderFunction(
```

Comment:
@wachterjohannes In #4101 I have introduced another variable in this namespace, so I change the structure a bit... I have extracted that part, and also renamed the file, as you have done in #3592. I hope @chirimoya merges this soon, then we have less conflicts later :-) Just FYI.